### ISSUE TITLE
how to translate the predict value to true depth?

### ISSUE THREAD

**Issue by ray-peng** [prstrive/EPCDepth#12]
hi! i used the pretrained model and got the predicted map. The values range from 0 to 1.3, what does this mean?

**Comment by prstrive** [comment]
We first estimate the disparity (range from 0 to 1) through the sigmoid function, and scale the disparity to depth with the range from 0.1 to 100. The monocular depth estimation model can only estimate the relative depth on the dataset with different camera parameters from the training dataset, and "The values range from 0 to 1.3" just means the relative depth, but not the actual depth value.

**Comment by ray-peng** [comment]
thanks a lot！

**Comment by carlinds** [comment]
Hi,
Can you elaborate a bit on this? Is it not possible to use the model for absolute depth estimation?
What are the use cases for relative depth estimation?
How do you evaluate your results against the ground truth if you can't estimate the absolute depth values?
I believe I might be misunderstanding your statement above.

**Comment by prstrive** [comment]
First of all, the training of monocular depth model if based on the image warping loss, which is jointly determined by depth and camera parameters. Therefore, the scale of the depth is related to the camera parameter, and you can predict the absolute depth if the camera parameter is the same as the training dataset, otherwise, you can only treat the estimated depth as the relative depth due to the ambiguity of the camera parameter. Second, of course there are many researches are proposed to study how to restore the absolute scale of monocular model through injecting some prior to the model. And you can check it out if you are interested.

**Comment by carlinds** [comment]
Thank you for taking the time to answer!
I better understand your original statement now, thanks. Your explanation was very helpful.
Do you have examples of the research you mentioned about injecting some prior to the model?

**Comment by prstrive** [comment]
Maybe you can refer to paper:
[1] Exploiting Pseudo Labels in a Self-Supervised Learning Framework for Improved Monocular Depth Estimation
[2] Toward hierarchical self-supervised monocular absolute depth estimation for autonomous driving applications
